flow/project.py

```diff
@@ -1490,6 +1493,26 @@ def run(self, jobs=None, names=None, pretend=False, np=None, timeout=None, num=N
             Show a progress bar during execution.
         :type progess:
             bool
+        :param order:
+            Specify the order of operations, possible values are:
+                * 'none' or None (no specific order)
+                * 'by-job' (operations are grouped by job)
+                * 'cyclic' (order operations cyclic by job)
+                * 'random' (explicitly shuffle the execution order randomly)
+                * callable (specify a key-callable used to sort operations)
```

Comment:
This description isn't too clear to me, mostly because `key-callable` isn't really a canonical term AFAIK. I would prefer to word this as something like `a callable returning a comparison key for an operation that is used to sort operations`, that matches the terminology used in the documentation of `sorted` (comparison key).